src/website/views/components/toolbar/_examples.pug

```diff
@@ -762,6 +762,406 @@ h2 Examples
           chi.dropdown(document.getElementById('example__base_with_ribbon_button_download'));
         </script>
 
+  h3 Base with ribbon and bulk actions
```

Comment:
As this example will be the same for all themes, why not take it outside? This will reduce duplicate code.

I would put it after `.-portal-hide` and `.-portal--show.-brightspeed--show` and remove it from `.-portal--show.-brightspeed--show`.